File: 30-days-of-code/java/Day5.java
import java.util.*;

class Day5 {
    public static void main(String[] args) {
        // create scanner
        Scanner in = new Scanner(System.in);

        // read input
        int input = in.nextInt();

        // close scanner
        in.close();

        // print multiplication table
        for (int i = 1; i < 11; i++)
            System.out.println(input + " x " + i + " = " + input * i);
    }
}

File: 30-days-of-code/csharp/Day5.cs
using System;

namespace Day5
{
    class Day5
    {
        static void Main(string[] args)
        {
            // read input
            int input = Convert.ToInt32(Console.ReadLine());

            // print multiplication table
            for (int i = 1; i < 11; i++)
                Console.WriteLine(input + " x " + i + " = " + input * i);
        }
    }
}
